tests/rbd/rbd_snap_clone_imported_image.py

```diff
@@ -0,0 +1,44 @@
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
```

Comment:
please update the doc strings as per this ref doc, https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

Reply:
Done